multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -168,6 +168,16 @@ const makeTestContext = async (t: ExecutionContext) => {
     makeTxOracle(oKeys[ix], { wd, vstorageClient, blockIter, now }),
   );
 
+  const attest = async (evidence: CctpTxEvidence, eudChain: string) => {
+    await Promise.all(
+      txOracles.map(async o => {
+        const { block } = await o.submit(evidence);
+        console.timeLog(`UX->${eudChain}`, o.getName(), block);
+      }),
+    );
+    console.timeLog(`UX->${eudChain}`, 'submitted x', txOracles.length);
+  };
+
   let callCount = 0;
```

Comment:
Unrelated idea:

We could initialize this with `agd query vstorage children published.fastUsdc.txns --output json | jq '.children | length'` so we don't get "tx already seen" across local test runs